src/client/js/Panels/MetaEditor/MetaEditorControl.js

```diff
@@ -460,6 +461,12 @@ define(['js/logger',
                     MetaRelations.META_RELATIONS.INHERITANCE);
             }
 
+            //MIXINS
+            len = this._nodeMixins[gmeID].length;
```

Comment:
What is the purpose to initialize the `len` here, if we overwrite it in the next statement?
It is a for loop call the iterator `i` instead.